beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -465,6 +477,11 @@ impl<TSpec: EthSpec> InboundRequest<TSpec> {
                 ProtocolId::new(Protocol::MetaData, Version::V2, Encoding::SSZSnappy),
                 ProtocolId::new(Protocol::MetaData, Version::V1, Encoding::SSZSnappy),
             ],
+            InboundRequest::LightClientBootstrap(_) => vec![ProtocolId::new(
+                Protocol::LightClientBootstrap,
+                Version::V1,
+                Encoding::SSZSnappy,
+            )],
```

Comment:
Inbound requests is where we need to ask about protocol support

Reply:
and what is the issue here? I see it is implemented accordingly? am i missing something?